kubetest/kubeadmdind/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
also add an OWNERS file for `kubeadmdind`

Reply:
@krzyzacy - Whom should I list as reviewers/approvers, and do we need labels in the OWNERS file?

Reply:
no labels are needed, and I would put yourself & people from your project as reviewers/approvers (don't need to put test-infra people here since we are already top-level approvers)

Reply:
Will add, thanks.